### DISCUSSION TITLE
What is your favorite proverb or saying? 好きなことわざや名言を教えてください！

### DISCUSSION THREAD

**Discussion by Yusuke196** [NAIST-SE/AttractingContributors#445] [General]
I like to see words of wisdom such as follows.

> Religion is a culture of faith; science is a culture of doubt. (Richard Feynman)

So please tell me a proverb or saying (ことわざや名言) you like by a PullRequest! My repo is [here](https://github.com/Yusuke196/AttractingContributors).

Note:

- Both English and Japanese are welcome. For words in other languages, I'd appreciate it if you add a translation.
- You don't necessarily need to write who said it.

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂

**Reply by Yusuke196** [reply]
Sure, but as you may have noticed, my PR is already merged in your repo!

**Comment by ploychanok** [comment]
Hi, I created a pull request to your repo.
Would you mind help mine too? https://github.com/ploychanok/AttractingContributors-WithinThreeWords
